### PR TITLE
use jssor for gallery

### PR DESCRIPTION
**Pull request by itkg-rboessel**
i removed choice of :
- respond time 
- number of columns
- items per page

I can add transition for each image

### CODE REVIEW

**Comment by itkg-rboessel** [comment]
@itkg-ngilain @nicolasThal @itkg-alavieille @itkg-nanne

File: MediaBundle/Resources/public/libs/jssor/jssor.js [hunk 1]
```diff
@@ -0,0 +1,2775 @@
+﻿/*
+* Jssor 19.0
```
Comment (nicolasThal) [inline MediaBundle/Resources/public/libs/jssor/jssor.js]: The jssor was not already set?

Comment (itkg-rboessel) [inline MediaBundle/Resources/public/libs/jssor/jssor.js]: No, I was an error because it was only jssor.slide.js but it need to be complete by jssor.js

**Comment by itkg-ngilain** [comment]
You can remove all the fancybox stuff (js libs) as it is no more used.

Also the code seems good, have you check to render a node with multiple gallery blocks to see if it works correctly, ie no js collision beetween differents galleries ?

**Comment by itkg-rboessel** [comment]
@itkg-ngilain it's not work with two gallery because  MediaBundle/Resources/public/block/Gallery/blockLoader.js
 is call multiple time and instanciate all galery each time.

**Comment by itkg-ngilain** [comment]
@itkg-rboessel : The problem is that the blockloader should not manage all matching blocks of the page, but only the one it was called for. That way you can add many times the twig, calling that many times the block loader, each time for a particular block.

You can try to pass the gallery_id to the block loader via a custom data-gallery-id attribute in the javascript tag. 

We can exchange via Slack or tel if you want on this point.

**Comment by itkg-rboessel** [comment]
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/11/files

File: MediaBundle/Resources/views/Block/Gallery/show.html.twig [hunk 2]
```diff
@@ -1,23 +1,30 @@
-<div id="{{galleryId}}" class="gallery {{ galleryClass }}">
-    <div>
-    {% for picture in pictures %}<a 
-        class="fancybox-thumb"
-        rel="fancybox-thumb"
-        href="{{ get_media_format_url(picture, imageFormat) }}"
-        title="{{ get_media_title(picture) }}"
-       ><img src="{{ get_media_format_url(picture, thumbnailFormat) }}" alt="{{ get_media_alt(picture) }}" class="gallery-picture" /></a>{% endfor %}
+<div id="{{ galleryId }}" class="jssor-gallery-container {{ galleryClass }}">
```
Comment (nicolasThal) [inline MediaBundle/Resources/views/Block/Gallery/show.html.twig]: you should put a protection to avoid id=""

**Comment by itkg-rboessel** [comment]
@nicolasThal change done

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**